recipes/hyper/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "hyper" %}
+{% set version = "0.7.0" %}
+{% set sha256 = "069514f54231fb7b5df2fb910a114663a83306d5296f588fffcb0a9be19407fc" %}
```

Comment:
Builds are getting `SHA256 mismatch`; looks like the right value is `12c82eacd122a659673484c1ea0d34576430afbe5aa6b8f63fe37fcb06a2458c`.